### DISCUSSION TITLE
umi4 ssr预计什么时候可以投入生产使用？

### DISCUSSION THREAD

**Discussion by weilun0510** [umijs/umi#9440] [Q&A]
umi4 ssr预计什么时候可以投入生产使用？现在使用的是umi4，是否有其他办法过渡一下，等待umi4 ssr投入生产。

**Comment by hinego** [comment]
真实无语。特么的这个umi4就是按揭发布

官网说的ssr / lowImport 什么的都没做好 就写上了。。。。。。我真的。。。

**Reply by weilun0510** [reply]
按揭哈哈哈哈，太cao了

**Reply by ShinChven** [reply]
鸿蒙式按揭开源，不丢人。

**Comment by citrusjunoss** [comment]
官方有提供时间节点之类的么，也在等ssr

**Comment by skitsanos** [comment]
I want SSR as well... in v3 it did very decent job, especially all together with Netlify Functions...

**Reply by ShinChven** [reply]
I just rolled back to v3 for SSR.
